open-metadata-implementation/server-chassis/server-chassis-spring/src/main/java/org/odpi/openmetadata/serverchassis/springboot/OMAGServerPlatform.java

```diff
@@ -98,10 +97,11 @@ public static void main(String[] args) {
     public InitializingBean getInitialize()
     {
         return () -> {
+            // TODO: This section will be removed once the new implementation is complete and tested
             if (!strictSSL)
             {
-                log.warn("strict.ssl is set to false! Invalid certificates will be accepted for connection!");
-                HttpHelper.noStrictSSL();
+                log.warn("strict.ssl is *DEPRECATED*! Please use egeria.ssl.client.verify system property or environment variable. If already set they will take precedence over strict.ssl. Note also that strict.ssl=true is the same as egeria.ssl.client.verify=false, and in the new implementation only affects client interaction");
```

Comment:
If you reach this point it is already set.
In the current implementation also affects only client interaction but on the JVM level which is more efficient

Reply:
Yes - noticed this last night.

JVM level didn't allow for the isolation (see above comment). Additionally I think it's important to try and keep our variables scoped clearly to egeria - hence the prefix 'egeria.xxxx' so that we don't get into any clashes with other applications . 'strict_ssl/strict-ssl' etc or a variant is used elsewhere. Same reason as above. Keep clarity for the egeria configuration.

I left support in in the short term to ensure things kept working